kmath-for-real/src/commonMain/kotlin/space/kscience/kmath/real/RealMatrix.kt

```diff
@@ -48,7 +48,7 @@ public fun Sequence<DoubleArray>.toMatrix(): RealMatrix = toList().let {
 }
 
 public fun RealMatrix.repeatStackVertical(n: Int): RealMatrix =
-    VirtualMatrix(rowNum * n, colNum) { row, col ->
+    VirtualMatrix( rowNum * n, colNum) { row, col ->
```

Comment:
Look like yet another typo.
```suggestion
    VirtualMatrix(rowNum * n, colNum) { row, col ->
```